### ISSUE TITLE
Dedicated Server: Lobby buttons not functional

### ISSUE THREAD

**Issue by Ryan089** [RE-SS3D/SS3D#992]
## Summary

The "Ready" and "Start Round" buttons within the Lobby appear to function when in dedicated server mode, but do not.

## Current Behavior

When clicked, the "Ready" button will appear to function correctly by changing to "Not Ready", and vice versa from "Not Ready" to "Ready". However, the visual change is the only actual effect, because a Broadcast cannot be sent from the server to itself. (A dedicated server may not even need a "Ready" button because it does not have a character to control - Centcom guidance requested on this). The "Start Round" button has the same behaviour - visual change but failed Broadcast.

## Steps to Reproduce

1. Set to server only mode by selecting **Edit > Project Settings > SS3D > Application Settings > Network Type = "Server Only"** and enter play mode. Alternately, to run a compiled build create a server .bat file within the /Builds directory using "**start SS3D.exe -serveronly -skipintro**" and execute the .bat file.
![image](https://user-images.githubusercontent.com/64360820/215732694-6eaf40ad-2361-4183-9bc3-f399151764cd.png)

2. Click on the "Ready" and "Start Round" buttons and observe their behaviour, including the absence of the round starting and presence of networking warnings.
![image](https://user-images.githubusercontent.com/64360820/215733304-7170d02e-0558-46fe-806c-a20b51796966.png)

![image](https://user-images.githubusercontent.com/64360820/215732332-ff209bbe-963f-42d1-af3f-99a3269bd04b.png)

## Expected Behavior

Buttons should be made functional, or removed if not required.

## Context

Applicable when running the game as a dedicated server, either in editor or with compiled build.

**Comment by stilnat** [comment]
@Ryan089 I agree, the ready button should be removed from the dedicated server.

**Comment by stilnat** [comment]
It would be even better if server only was headless, not showing any kind of UI but a console.
